### PR TITLE
Core 957 qf channel improve page initialization

### PR DESCRIPTION
**Pull request by dyersituations**
https://kiva.atlassian.net/browse/CORE-957
https://kiva.atlassian.net/browse/CORE-897

- Quick filters now enabled for all categories (3 used to be disabled)
- Category pages again use cached loan data (since the filters don't currently change on mount)
- Because prefetched data is being used again, the page no longer initially loads with "no loans found"

### CODE REVIEW

**Review by eddieferrer — APPROVED** [review]

**Review by roger-in-kiva — APPROVED** [review]
Looks good!

File: src/pages/Lend/LoanChannelCategoryControl.vue [hunk 1]
```diff
@@ -2,9 +2,7 @@
 	<div class="tw-relative">
 		<div class="row">
 			<div class="small-12 columns heading-region">
-				<view-toggle v-if="!enableQuickFilters" browse-url="/lend-by-category" :filter-url="filterUrl" />
```
[the remark this answers is not in the document]
Comment (dyersituations) [reply src/pages/Lend/LoanChannelCategoryControl.vue:5]: Good question, and I have no idea. I'll ask in Denali channel before merging this PR.

**Review by mcstover — APPROVED** [review]
Thanks for helping solidify this! Looking forward to the additional cleanup coming soon too!

File: src/pages/Lend/LoanChannelCategoryControl.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (dyersituations) [reply src/pages/Lend/LoanChannelCategoryControl.vue:5]: Checked with design, and we want to keep the new "advanced filters" link on the category pages, due to good conversion results from the QF + help me choose experiment, that included this update link.

**Pull request merged**